docs/src/PageHeader.doc.js

```diff
@@ -0,0 +1,643 @@
+        Use subtext to add a description about the page. It should only be used for metadata.
+        `}
+        defaultCode={`
+<PageHeader
+  title="Product groups"
+  subtext="Product groups are created in order to relate certain products together for tracking purposes"
+  primaryAction={<Button color="red" size="lg" text="Create product group" />}
+  secondaryAction={<Button text="Promote" size="lg" />}
+/>;
+        `}
+        shaded
+      />
+      <MainSection.Card
+        cardSize="lg"
+        type="don't"
+        description={`
+        Use \`maxWidth\` when the content of the page is not center aligned.
+        `}
+        defaultCode={`
+<Flex gap={4} direction="column" flex="grow">
+  <PageHeader
+    title="Settings"
+    maxWidth="50%"
+    primaryAction={
+      <Tooltip text="Additional options" idealDirection="up">
+        <IconButton icon="ellipsis" iconColor="darkGray" size="lg" accessibilityLabel=""/>
+      </Tooltip>
+    }
+  />
+  <Box
+    display="flex"
+    wrap
+    width="80%"
+    direction="column"
+    marginStart={5}
+  >
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <Heading size="sm" accessibilityLevel={2}>Edit profile</Heading>
+    </Box>
+
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <TextField
+        label="TextField 4"
+        id="textfield4"
+        onChange={() => {}}
+        placeholder="Placeholder"
+      />
+    </Box>
+
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <Box display="flex" wrap marginStart={-3} marginEnd={-3} marginBottom={-3} marginTop={-3}>
+        <Box flex="grow" paddingX={3} paddingY={3}>
+          <TextField
+            label="TextField 5"
+            id="textfield5"
+            onChange={() => {}}
+            placeholder="Placeholder"
+          />
+        </Box>
+        <Box flex="grow" paddingX={3} paddingY={3}>
+          <TextField
+            label="TextField 6"
+            id="textfield6"
+            onChange={() => {}}
+            placeholder="Placeholder"
+          />
+        </Box>
+      </Box>
+    </Box>
+
+    <Box flex="grow" paddingX={3} paddingY={3}>
+      <SelectList
+        label="SelectList"
+        id="selectlist0"
+        options={[
+          {
+            value: 'belgium',
+            label: 'Belgium',
+          },
+          {
+            value: 'france',
+            label: 'France',
+          },
+          {
+            value: 'usa',
+            label: 'USA',
+          },
+        ]}
+        placeholder="Placeholder"
+        onChange={() => {}}
+      />
+    </Box>
+  </Box>
+</Flex>;
+        `}
+        shaded
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Accessibility"
+    description={`Be sure to follow any accessibility guidelines for the components used within PageHeader. The heading within PageHeader will be rendered as a level 1 heading, so ensure there are no other level 1 headings present on the page. `}
```

Comment:
Could we add here some Resource links? Probably not, just to make it a little bit more juicy.

Reply:
Maybe an example in combination with Heading 2, 3 could help here. Also, what about adding See Related to learn more about the Heading and HeaderPage. Or maybe we could add `Heading should be used to create level 2-6 headings on a page. If a level 1 heading is needed, use PageHeader` here as well as it's pretty connected.